### PR TITLE
#67 Your bill text overlap

### PR DESCRIPTION
**Pull request by VDuda**
- "Your Bill" text will be in the middle of the bar
- Refactored spacing
- Added vim swp to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore vim swaps
+*.swp
```
Comment (mzagaja) [inline .gitignore:22]: Please put these kind of ignores in your global .gitignore. See https://help.github.com/articles/ignoring-files/ for more info.

**Comment by mzagaja** [comment]
I tested this out on my local machine and I think this works well, at least until we think of a better way to deal with it. Delete the .gitignore from the pull request and I'll merge this into master. Thanks for your help!

**Comment by mzagaja** [comment]
Any chance you can try something more like http://jsfiddle.net/MmZM9/?

**Comment by itiswicked** [comment]
closes #67

**Comment by mzagaja** [comment]
@itiswicked FYI I was testing it last week and the "closes #" convention seems to work only in commit messages and not in comments on the GitHub website (although it will make a reference). I think because the comments on the pull request thread don't actually get brought into the git history, they just live on GitHub's website.

**Comment by itiswicked** [comment]
Ahh, good to know. Thanks!

**Comment by mzagaja** [comment]
Since I want to take a different direction with this and haven't heard back I'm going to close this.

**Pull request closed without merging**